master.sh

```diff
@@ -40,13 +40,7 @@ init() {
 	fi
 
 	# Set working mode
-	if [[ -n ${START_MODE} && ${START_MODE} == "install" ]]; then
-		echo "Start k8s cluster in installation mode"
 		RESTART_POLICY="always"
```

Comment:
@zreigz @cheld I thought the start mode for the scripts was supposed to be either as a `systemd service` or as a `hyperkube` container. Do we additionally want to support different restart policies for `hyperkube`? If so I can easily revert my changes, but we'll need one more additional parameter passed.